recipes-browser/wpebackend-fdo/wpebackend-fdo_1.8.4.bb

```diff
@@ -3,5 +3,5 @@ require wpebackend-fdo.inc
 inherit cmake
 
 SRC_URI = "https://wpewebkit.org/releases/${BPN}-${PV}.tar.xz"
```

Comment:
This could be in `wpebackend-fdo.inc` as well, and using `${P}` instead of `${BPN}-${PV}` 😉